### PR TITLE
Pre-release v4.0.0

### PR DESCRIPTION
**Pull request by lucianomartin**
* Update AVS Device SDK version to version 3.0.0
  * Update vocalfusion-rpi-setup to version 5.8.0
  * Removed support for Sensory KWD
  * Removed support XMOS AVS tests
  * Removed code to control LED's of XVF3510 Pi hat
  * Removed support for Vocalfusion sales demo

Part of #69
Part of #74

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +38,8 @@ You will need:
 
 - [for -UA configurations] USB A to USB Micro B cable
 
+We do not advice using a Raspberry Pi 3, in place of a Raspberry Pi 4. Lack of RAM on the Raspberry Pi 3 may cause the AVS device SDK compilation to fail, as reported in [this issue](https://github.com/alexa/avs-device-sdk/issues/2035).
```
Comment (mbanth) [inline README.md:41]: `advice` -> `advise`.  Advice is a noun while advise is a verb.

**Review by mbanth — CHANGES_REQUESTED** [review]
One small change

File: README.md (hunk 1 above)
Comment (lucianomartin) [reply README.md]: Updated

**Review by mbanth — APPROVED** [review]
Looks good now.

**Pull request merged**